portuguese.json

```diff
@@ -0,0 +1,119 @@
+    "forms.teleporter": "Teleporter",
+    "forms.teleporter.full": "CHEIO",
+    "forms.teleporter.offline": "OFFLINE",
+    "forms.teleporter.partly_online": "PARTLY ONLINE",
+    "forms.teleporter.online": "ONLINE",
+    "forms.teleporter.select": "Selecione o minigame:",
+    "join.vote": "Vote em nós diariamente em §§angmc.co/v §§me pode ganhar uma melhoria de rank gratuita!",
+    "join.welcome": "Bem-vindo ao §eNether§6Games§§m!",
+    "menu.blocked": "Não tem permissão para usar esse menu.",
+    "party.already.in.host": "{%invited} já está numa party!",
+    "party.already.invited": "Você já convidou {%invited}.",
+    "party.ended.host": "Você finalizou a sua party.",
+    "party.invited": "Convidou {%invited} para a sua party.",
+    "party.join.game": "Você esta agora em uma party. Espere que o dono da sua party decida qual o jogo a jogar!",
+    "party.kicked": "Foi expulso da party por {%kicker}.",
+    "party.kicked.message": "{%kicker} expulsou {%kicked} da party.",
+    "party.leave.player": "Você abandonou a party.",
+    "party.max": "Você não pode convidar mais do que {%max} jogadores para a sua party!",
+    "party.still.in.game": "{%member} ainda está em jogo. Espere por eles antes de se juntar a outro!",
+    "party.welcome": "Bem-vindo a party de {%host}",
+    "pets.ride.noperm": "Não tem permissão para montar e controlar pet de estimação. Compre o §l§cTITAN§r §§mrank em §§angmc.co/store §§mno site!",
+    "player.offline": "Esse jogador não está online.",
+    "staff.impersonating": "Não é permitido se fazer passar por membros do staff.",
+    "vote.claimed": "Já votou e reivindicou hoje as suas recompensas.",
+    "vote.error": "Ocorreu um erro desconhecido durante a verificação do seu estado de voto.",
+    "vote.notyet": "Ainda não votou hoje. Vote em §§angmc.co/v§§m, depois volte e receba as suas recompensas.",
+    "vote.successful": "Reivindicou com sucesso as suas recompensas. Obrigado por votar em §eNether§6Games§§m!",
+    "secret.unlock.all": "Parabéns, completou os segredos do lobby! Que aventura selvagem foi esta. Aproveite o seu tempo, e esperamos que tenha gostado dos pequenos puzzles :)",
+    "zone.fly.disabled": "O modo de voo é desativado nesta zona",
+    "secret.unlocked.title": "Segredo desbloqueado",
+    "crate.inuse": "Esta caixa está sendo utilizada no momento",
+    "fireball.perfect.hit": "Você golpeou {%playerName} com {%blocks} blocos!",
+    "fireball.perfect.xp": "+{%xp} XP ({%perfectShot} tiro(s) perfeito(s) com fireball)",
+    "skywars.luckyblock.no_chests": "baús desativados no lucky block!",
+    "skywars.cooldown.item": "{%item} fica em tempo de espera durante {%seconds} segundos!",
+    "skywars.setup.add_middle_chest": "Adicionado com sucesso o peitoral médio em {%position}!",
+    "skywars.setup.enter_middle_chest_setup_mode": "Modo de configuração do baú para: {%arena}\nTap baús para adicionar ao baú central ou usar comando '/sw editar set do baú de id para parar.",
```

Comment:
This translation is incorrect. The original string is "Entered chest configuration mode for: {%arena}\nTap chests to add to middle chests or use command '/sw edit setmidchest' to stop."

Reply:
Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).